app/views/layouts/application.html.slim

```diff
@@ -1,5 +1,5 @@
 doctype html
-html lang=I18n.locale data-default-locale=I18n.default_locale
+html lang=I18n.locale *html_data_attributes
```

Comment:
After we got the other parts right here, I am returning to my previous question regarding the gloabl scope. 

Previously, you wrote:

> The JavaScript needs this when the page loads. I don't want to make too large changes to JavaScript at the moment.

Some of the data attributes are not needed when the page loads, but can also be parsed later, I'd say. Rather than touching this part multiple times, I would suggest to get it right now.